app.py

```diff
@@ -44,11 +49,12 @@ def require_login(user_param_name='user'):
     def inner_decorator(func):
         @wraps(func)
         def wrapped_func(*args, **kwargs):
-            token = None
-            if request.method == 'GET':
-                token = request.args.get('token')
-            elif request.method == 'POST':
-                token = request.values.get('token')
+            token = request.args.get('token')
+            if token is not None:
+                resp = redirect(request.base_url)
+                resp.set_cookie('token',token,httponly=True,samesite='lax')
```

Comment:
Cookie should be `secure` as well.

Reply:
Can this be fixed in a followup PR? @brynolv make an issue for this and we can merge it.

Reply:
I made an issue #16